File: esportskicentarapp/src/app/models/prihodi-rashodi-stats-dto.ts
export class PrihodiRashodiStatsDto {
    datum: string;
    prihodi: number;
    rashodi: number;

    constructor(datum: string, prihodi: number, rashodi: number) {
        this.datum = datum;
        this.prihodi = prihodi;
        this.rashodi = rashodi;
    }
}

File: esportskicentar/src/main/java/tech/esc/esportskicentar/model/PrihodiRashodiStatsDTO.java
package tech.esc.esportskicentar.model;

import lombok.*;

import java.math.BigDecimal;

@Data
@EqualsAndHashCode(callSuper = false)
@NoArgsConstructor
@AllArgsConstructor
@Builder
public class PrihodiRashodiStatsDTO {
    private BigDecimal prihodi;
    private BigDecimal rashodi;
}
